docker/soarca/docker-compose.caldera.yml

```diff
@@ -0,0 +1,9 @@
+
+services:
+  soarca:
+    environment:
+      CALDERA_URL: "http://caldera:8888"
+    networks:
+      - caldera-net
+    depends_on:
+      - caldera
```

Comment:
I did deliberately put this in a separate file (see also the docs). This allows the user to choose if they do or do not want to use Caldera, without editing Compose files.

Reply:
I would then add the caldera service itself here as well, so it does not run when not specified.

Reply:
https://docs.docker.com/compose/how-tos/profiles/

Reply:
Excellent suggestion @MrDurion , implemented